lib/Traits/TraitsClasses/WebFeatureServiceCatalogItemTraits.ts

```diff
@@ -25,6 +32,22 @@ export default class WebFeatureServiceCatalogItemTraits extends mixTraits(
   })
   maxFeatures = 1000;
 
+  @primitiveTrait({
+    type: "string",
+    name: "Srs Name",
+    description: `Spatial Reference System to use. For WFS we prefer WGS 84 (${SUPPORTED_CRS_4326.join(
+      ", "
+    )}). With WFS requests it is best to use the urn identifier for the srsName, to enforce lat,long order in returned results.`
+  })
+  srsName?: string;
+
+  @primitiveTrait({
+    type: "string",
+    name: "Output Format",
+    description: `Output format to request for WFS requests. We support gml3 and gml3.1.1.`
```

Comment:
It might be worth adding that we support GeoJSON - and saying we only support GML in 4326